google-cloud-storage/src/main/java/com/google/cloud/storage/transfermanager/TransferManagerImpl.java

```diff
@@ -56,6 +56,14 @@ public TransferManagerImpl(TransferManagerConfig transferManagerConfig) {
 
   @Override
   public @NonNull DownloadJob downloadBlobs(List<BlobInfo> blobs, ParallelDownloadConfig opts) {
```

Comment:
Docs?

Reply:
docs will be a followup, just getting first passes out first.